### ISSUE TITLE
Seeing new issue with newer ansible

### ISSUE THREAD

**Issue by harlowja** [tomoh1r/ansible-vault#6]
Seeing the following now happen (maybe ansible changed the api?):

```
>>> utils.load_secrets("secrets/daddy.eyaml")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/jxharlow/Dev/els/openstack-deploy/os_deploy/utils.py", line 87, in load_secrets
    return munch.munchify(v.load(fh.read()))
  File "/Users/jxharlow/.venv/lib/python2.7/site-packages/ansible_vault/api.py", line 18, in load
    return yaml.safe_load(self.vault.decrypt(stream))
  File "/Users/jxharlow/.venv/lib/python2.7/site-packages/ansible/parsing/vault/__init__.py", line 543, in decrypt
    _matches = match_secrets(self.secrets, vault_id_matchers)
  File "/Users/jxharlow/.venv/lib/python2.7/site-packages/ansible/parsing/vault/__init__.py", line 399, in match_secrets
    matches = [(vault_id, secret) for vault_id, secret in secrets if vault_id in target_vault_ids]
ValueError: need more than 1 value to unpack
```

**Comment by harlowja** [comment]
Ansible 2.4.0 may have caused some funkyiness here... just fyi.

**Comment by enrh** [comment]
I'm also having this issue.

**Comment by brokenjacobs** [comment]
same here. my dynamic inventory is broken with 2.4.0

**Comment by tomoh1r** [comment]
Hi there, Thanks for your information. I'll investigate it.

**Comment by tomoh1r** [comment]
Fix it and [release 1.1.0 to pypi](https://pypi.python.org/pypi/ansible-vault/1.1.0). Check it out.

**Comment by tomoh1r** [comment]
#7
